### PR TITLE
Update WindowsBuild.md

### PR DESCRIPTION
**Pull request by compnerd**
Update for the monorepo.

Replace this paragraph with a description of your changes and rationale. Provide links to external references/discussions if appropriate.

Resolves SR-NNNN.

### CODE REVIEW

File: docs/WindowsBuild.md [hunk 1]
```diff
@@ -8,20 +8,10 @@ Visual Studio 2017 or newer is needed to build swift on Windows.
...
-1. Clone `apple/swift-llvm` into a folder named `llvm`
-1. Clone `apple/swift-compiler-rt` into a folder named `compiler-rt`
-1. Clone `apple/swift` into a folder named `swift`
-1. Clone `apple/swift-corelibs-libdispatch` into a folder named `swift-corelibs-libdispatch`
-1. Clone `apple/swift-corelibs-foundation` into a folder named `swift-corelibs-foundation`
-1. Clone `apple/swift-corelibs-xctest` into a folder name `swift-corelibs-xctest`
-1. Clone `apple/swift-lldb` into a folder named `lldb`
-1. Clone `apple/swift-llbuild` into a folder named `llbuild`
-1. Clone `apple/swift-package-manager` info a folder named `swift-package-manager`
+## 1. Clone the repositories
```
Comment (CodaFi) [inline docs/WindowsBuild.md]: Nit: Does it make sense to number the headers like this?  Most of them appear to be 1 now...

**Comment by compnerd** [comment]
@swift-ci please smoke test

File: docs/WindowsBuild.md (hunk 1 above)
Comment (compnerd) [reply docs/WindowsBuild.md]: Yeah, I was hoping that markdown would be smart and renumber them.  I think that I will replace them and remove the numbering in a subsequent change.

**Comment by compnerd** [comment]
@swift-ci please smoke test macOS platform

**Comment by compnerd** [comment]
@swift-ci please smoke test macOS platform

**Comment by compnerd** [comment]
@swift-ci please test macOS platform

**Comment by compnerd** [comment]
@swift-ci please smoke test macOS platform

**Comment by compnerd** [comment]
There is more work that needs to be done to improve the docs, but this is a step in the right direction.

**Pull request merged**